lib/libvmod_proxy/vmod.vcc

```diff
@@ -1,8 +1,10 @@
 #-
 # Copyright (c) 2018 GANDI SAS
+# Copyright 2018 UPLEX - Nils Goroll Systemoptimierung
```

Comment:
"Copyright (c)" for consistency?

Reply:
I once learned that *Copyright (c)* actually is wrong. No idea, but I would like to stick to it, each their own :)